<launch>

	<arg name="raw_lidar_topic" default="/airsim_ros_node/MyVehicle/lidar/Lidar2"/>
	<arg name="input_cloud_type" default="XYZIT"/>
	<arg name="output_cloud_type" default="XYZIR"/>
	
	<node pkg="rs_to_velodyne" exec="rs_to_velodyne" name="rs_to_velodyne" output="screen">
        <param name="raw_lidar_topic" value="$(var raw_lidar_topic)"/>
        <param name="input_cloud_type" value="$(var input_cloud_type)"/>
        <param name="output_cloud_type" value="$(var output_cloud_type)"/>
    </node>

</launch>
